pkg/agent/controller/traceflow/packetin_test.go

```diff
@@ -53,6 +56,59 @@ func (f *fakeEgressQuerier) GetEgressIPByMark(mark uint32) (string, error) {
 	return f.egressIP, nil
 }
 
+type fakeAgentNetworkPolicyInfoQuerier struct {
```

Comment:
We also have mocked AgentNetworkPolicyInfoQuerier, you can use `NewMockAgentNetworkPolicyInfoQuerier` to defined a mocked querier and add expectedCalls in the tests struct.

Reply:
done, thanks.